File: wrappers/mvc/demos/Kendo.Mvc.Examples/Controllers/DataViz/Polar_Charts/Smooth_Polar_LineController.cs
using System;
using System.Linq;
using System.Web.Mvc;
using Kendo.Mvc.Examples.Models;

namespace Kendo.Mvc.Examples.Controllers
{
    public partial class Polar_ChartsController : Controller
    {
        public ActionResult Smooth_Polar_Line()
        {
            return View(ChartDataRepository.SunPosition());
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/polarcharts/SmoothPolarLineController.java
package com.kendoui.spring.controllers.polarcharts;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

import com.kendoui.spring.models.ChartDataRepository;

@Controller("dataviz-polar_chart-smooth_polar_line-controller")
@RequestMapping(value="/polar-charts/")
public class SmoothPolarLineController {
    @RequestMapping(value = "/smooth-polar-line", method = RequestMethod.GET)
    public String index(Model model) {
        model.addAttribute("sunPosition", ChartDataRepository.SunPositionData());
     
        return "/polar-charts/smooth-polar-line";
    }
}
